hack/lib/golang.sh

```diff
@@ -836,7 +836,7 @@ kube::golang::build_binaries_for_platform() {
    done
 
   V=2 kube::log::info "Env for ${platform}: GOPATH=${GOPATH-} GOOS=${GOOS-} GOARCH=${GOARCH-} GOROOT=${GOROOT-} CGO_ENABLED=${CGO_ENABLED-} CC=${CC-}"
-  V=3 kube::log::info "Building binaries with GCFLAGS=${gogcflags} LDFLAGS=${goldflags}"
+  V=3 kube::log::info "Building binaries with GCFLAGS=${gogcflags} LDFLAGS=${goldflags} and -tags=${gotags:-}"
```

Comment:
this still works! try with `KUBE_VERBOSE=4 make kubelet` and you will see `-tags=selinux,notest,grpcnotrace,` in the output